bee-api/bee-rest-api/src/types/dtos.rs

```diff
@@ -536,6 +675,478 @@ pub struct ReferenceUnlockBlockDto {
+            FeatureBlock::ExpirationMilestoneIndex(v) => {
+                Self::ExpirationMilestoneIndex(ExpirationMilestoneIndexFeatureBlockDto(v.index()))
+            }
+            FeatureBlock::ExpirationUnix(v) => Self::ExpirationUnix(ExpirationUnixFeatureBlockDto(v.timestamp())),
+            FeatureBlock::Indexation(v) => Self::Indexation(IndexationFeatureBlockDto(v.to_string())),
+            FeatureBlock::Metadata(v) => Self::Metadata(MetadataFeatureBlockDto(v.to_string())),
+        }
+    }
+}
+
+impl TryFrom<&FeatureBlockDto> for FeatureBlock {
+    type Error = Error;
+
+    fn try_from(value: &FeatureBlockDto) -> Result<Self, Self::Error> {
+        Ok(match value {
+            FeatureBlockDto::Sender(v) => Self::Sender(SenderFeatureBlock::new((&v.0).try_into()?)),
+            FeatureBlockDto::Issuer(v) => Self::Issuer(IssuerFeatureBlock::new((&v.0).try_into()?)),
+            FeatureBlockDto::DustDepositReturn(v) => Self::DustDepositReturn(DustDepositReturnFeatureBlock::new(v.0)?),
+            FeatureBlockDto::TimelockMilestoneIndex(v) => {
+                Self::TimelockMilestoneIndex(TimelockMilestoneIndexFeatureBlock::new(v.0))
+            }
+            FeatureBlockDto::TimelockUnix(v) => Self::TimelockUnix(TimelockUnixFeatureBlock::new(v.0)),
+            FeatureBlockDto::ExpirationMilestoneIndex(v) => {
+                Self::ExpirationMilestoneIndex(ExpirationMilestoneIndexFeatureBlock::new(v.0))
+            }
+            FeatureBlockDto::ExpirationUnix(v) => Self::ExpirationUnix(ExpirationUnixFeatureBlock::new(v.0)),
+            FeatureBlockDto::Indexation(v) => Self::Indexation(IndexationFeatureBlock::new(
+                hex::decode(&v.0).map_err(|_e| Error::InvalidSemanticField("IndexationFeatureBlock"))?,
+            )?),
+            FeatureBlockDto::Metadata(v) => Self::Metadata(MetadataFeatureBlock::new(
+                hex::decode(&v.0).map_err(|_e| Error::InvalidSemanticField("MetadataFeatureBlock"))?,
+            )?),
+        })
+    }
+}
+
+/// Describes an alias account in the ledger that can be controlled by the state and governance controllers.
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct AliasOutputDto {
+    #[serde(rename = "type")]
+    pub kind: u8,
+    // Amount of IOTA tokens held by the output.
+    pub amount: u64,
+    // Native tokens held by the output.
+    pub native_tokens: Vec<NativeTokenDto>,
+    // Unique identifier of the alias.
+    pub alias_id: AliasIdDto,
+    //
+    pub state_controller: AddressDto,
+    //
+    pub governance_controller: AddressDto,
+    // A counter that must increase by 1 every time the alias is state transitioned.
+    pub state_index: u32,
+    // Metadata that can only be changed by the state controller.
+    pub state_metadata: String,
+    // A counter that denotes the number of foundries created by this alias account.
+    pub foundry_counter: u32,
+    //
+    pub feature_blocks: Vec<FeatureBlockDto>,
```

Comment:
And here it needs to be `#[serde(rename = "blocks")]` if it's the same as for the extended output

Reply:
Changed that! thanks